mesonbuild/interpreter.py

```diff
@@ -916,16 +919,30 @@ def get_name(self):
 
 class SubprojectHolder(InterpreterObject, ObjectHolder):
 
-    def __init__(self, subinterpreter):
+    def __init__(self, subinterpreter, subproject_dir, name):
         InterpreterObject.__init__(self)
         ObjectHolder.__init__(self, subinterpreter)
+        self.name = name
+        self.subproject_dir = subproject_dir
         self.methods.update({'get_variable': self.get_variable_method,
+                             'found': self.found_method,
                              })
 
+    @noPosargs
+    @permittedKwargs({})
+    def found_method(self, args, kwargs):
+        return self.found()
+
+    def found(self):
+        return self.held_object is not None
+
```

Comment:
I think these are no longer required now that we don't try to use a `SubprojectHolder` as a `Dependency`. @thiblahute could you open a PR that removes these?

Reply:
Well, at least `found_method()` is not needed. :)

Reply:
Well, it is... so the user can know if the subproject is usable or not?